src/System Application/Test/DisabledTests/AFSFileClient.json

```diff
@@ -0,0 +1,8 @@
+[
+    {
+        "bug":  "",
+        "codeunitId":  132517,
```

Comment:
Why are the tests disabled? Does that mean we've got 0% test coverage?

Reply:
I think the problem was, when the tests were created, that @darjoo didn't allow any IsHandled pattern for Testing Purpose only.

Therefore those tests require a real Azure File Share Instance for testing.

Azurite doesn't support Azure File Share yet: https://github.com/Azure/Azurite/issues/113

The module maybe needs to be refactored to use the RestClient and give the possibility to specify a HttpClientHanlder.  -> At least for testing purpose. Then it might be possible to Mock Web Requests.